sphinx/user/getting_started.rst

```diff
@@ -251,12 +251,12 @@ Now for using the pipeline.
   simply replace ``visit=58`` with ``arm=r spectrograph=2 dateObs=2015-12-19
   field=ARC``::
 
-     detrend.py $PFS_DATA --rerun $whoami/tmp --id visit=58 -c isr.doBias=True isr.doDark=True isr.doFlatTrue isr.doLinearize=False
+     detrend.py $PFS_DATA --rerun $whoami/tmp --id visit=58 -c isr.doBias=True isr.doDark=True isr.doFlat=True isr.doLinearize=False
```

Comment:
Do we need these config overrides?  Please update config/pfs/detrend.py in obs_pfs and remove them here.